ucfs-claimant-get-award-details.tf

```diff
@@ -0,0 +1,34 @@
+resource "github_repository" "ucfs-claimant-get-award-details" {
+  name        = "ucfs-claimant-get-award-details"
+  description = "Get Award Details Lambda code for UCFS Claimant API endpoint"
+  auto_init   = true
```

Comment:
This will cause the repo to be recreated (but prevent_destroy will block it). It can only be used for new repos as per the readme

Reply:
it's a new repo

Reply:
Ok. Confused without access to concourse then to see where the related release artifacts currently come from.